From e2d95bd0fa0f9eb5236cf5273dc5fa1f1ec4a72b Mon Sep 17 00:00:00 2001
From: hhb <hhb@rock-chips.com>
Date: Mon, 11 Jul 2011 18:06:29 +0800
Subject: [PATCH] Revert "rk29:fix bug arch/arm/mach-rk29/board-rk29-a22.c:493:
 warning: missing braces around initializer"

This reverts commit 589de6220192ac431424627747bade4c391f6363.
---
 arch/arm/mach-rk29/board-rk29-phonesdk.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-rk29/board-rk29-phonesdk.c b/arch/arm/mach-rk29/board-rk29-phonesdk.c
index 10d88300fb02..ea480afb353f 100644
--- a/arch/arm/mach-rk29/board-rk29-phonesdk.c
+++ b/arch/arm/mach-rk29/board-rk29-phonesdk.c
@@ -465,7 +465,7 @@ static struct gt801_platform_data gt801_info = {
 	.gpio_reset_active_low = 0,
 	.gpio_pendown		= GT801_GPIO_INT,
     .pendown_iomux_name = GPIO4D5_CPUTRACECTL_NAME,
-    .resetpin_iomux_name = "",
+    .resetpin_iomux_name = NULL,
     .pendown_iomux_mode = GPIO4H_GPIO4D5,
     .resetpin_iomux_mode = 0,
 };
@@ -487,7 +487,7 @@ static struct gt818_platform_data gt818_info = {
 	.gpio_reset_active_low = 0,
 	.gpio_pendown		= GT818_GPIO_INT,
     .pendown_iomux_name = GPIO4D5_CPUTRACECTL_NAME,
-    .resetpin_iomux_name = "",
+    .resetpin_iomux_name = NULL,
     .pendown_iomux_mode = GPIO4H_GPIO4D5,
     .resetpin_iomux_mode = 0,
 };
-- 
2.35.3

